assets/airports/kpvd.json

```diff
@@ -0,0 +1,6570 @@
+        "YODER":    ["N41d17m22.5100",  "W072d20m32.7900"],
+        "ZADOR":    ["N41d39m56.9700",  "W072d48m00.3200"],
+        "ZAGAR":    ["N42d00m21.2300",  "W072d03m24.0800"],
+        "ZARAS":    ["N41d43m56.6700",  "W072d14m16.3100"],
+        "ZEPVI":    ["N41d34m33.7300",  "W072d52m43.9100"],
+        "ZOVOX":    ["N41d42m13.0600",  "W072d36m52.9500"]
+    },
+    "runways":[
+        {
+            "name": ["05", "23"],
+            "end": [
+                ["N41d42m49.7723",  "W071d26m05.8486", "50.8ft"],
+                ["N41d43m49.6961",  "W071d25m15.5354", "44.3ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["16", "34"],
+            "end":  [
+                ["N41d43m53.9542",  "W071d25m55.8304", "53.7ft"],
+                ["N41d43m06.8846",  "W071d25m05.9925", "32.8ft"]
+            ],
+            "ils": [false, true]
+        }
+    ],
+    "sids": {
+        "BAF": {
+            "icao": "BAF",
+            "name": "Barnes",
+            "rwy": {
+                "05": "",
+                "23": "",
+                "16": "",
+                "34": ""
+            },
+            "body": ["PUT", "NELIE"],
+            "exitPoints": {
+                "BAF": "BAF"
+            },
+            "draw": [["PUT", "NELIE", "BAF"]]
+        },
+        "JUMPR": {
+            "icao": "JUMPR",
+            "name": "Jumper",
+            "rwy": {
+                "05": "",
+                "23": "",
+                "16": "",
+                "34": ""
+            },
+            "body": ["NICOD", "FALMA"],
+            "exitPoints": {
+                "JUMPR": ["JUMPR"]
+            },
+            "draw": [["NICOD", "FALMA", "JUMPR"]]
+        }
+    },
+    "stars": {
+        "JORDN2": {
+            "icao": "JORDN2",
+            "name": "Jordan Two",
+            "entryPoints": {
+                "HTO": ["HTO", "JORDN"]
+            },
+            "body": ["MINNK"],
+            "draw": [["HTO*", "JORDN", "MINNK"]]
+        },
+        "WIPOR3": {
+            "icao": "WIPOR3",
+            "name": "Wipor Three",
+            "entryPoints": {
+                "ALB": ["ALB", ["SHIGY", "A180"]],
+                "HNK": ["HNK", ["HINGZ", "A290"]],
+                "HINGZ": [["HINGZ", "A290"]]
+            },
+            "body": ["PONEE", ["NELIE", "A210"], ["WIPOR", "A110"], "ORW", "SAUGY"],
+            "rwy": {
+                "05": ["LAFAY"],
+                "34": ["LAFAY"],
+                "16": ["WAWIK", "SEABE", "NOSUH"],
+                "23": ["WAWIK", "SEABE", "NOSUH"]
+            },
+            "draw": [
+                ["ALB*", "SHIGY", "PONEE"],
+                ["HNK*", "HINGZ*", "PONEE"],
+                ["PONEE", "NELIE", "WIPOR", "ORW", "SAUGY"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "KPVD.BAF.BAF",
```

Comment:
and then all these can just be changed to `KPVD..BAF`, provided that `KPVD` is a defined fix.